packages/api/apollo-server/types.ts

```diff
@@ -1,13 +1,82 @@
 import { gql } from 'apollo-server-core'
 
 const Types = gql`
+  input UserInput {
+    id: String
+    firstName: String
+    lastName: String
+    utorID: String
+    userType: UserType
+  }
+
   type Query {
-    getUser: User
-    getUsers: [User]
+    getUser(id: String, utorID: String, email: String): User
+    getUsers(users: [UserInput!]!): [User!]!
+    getAssignment(id: String): Assignment
```

Comment:
Should always return an Assignment. Also, since id is the only field, it must be required.